packages/strapi-generate-plugin/files/admin/src/containers/Initializer/tests/index.test.js

```diff
@@ -0,0 +1,25 @@
+// import React from 'react';
+// import { mount, shallow } from 'enzyme';
+// import pluginId from '../../pluginId';
+
+// import Initializer from '../index';
+
+describe('<Initializer />', () => {
+  // it('Should not crash', () => {
+  //   const updatePlugin = jest.fn();
+  //   const renderedComponent = shallow(<Initializer updatePlugin={updatePlugin} />);
+
+  //   expect(renderedComponent.children()).toHaveLength(0);
+  // });
+
+  // it('should call the updatePlugin props when mounted', () => {
+  //   const updatePlugin = jest.fn();
+
+  //   const wrapper = mount(<Initializer updatePlugin={updatePlugin} />);
+
+  //   expect(wrapper.prop('updatePlugin')).toHaveBeenCalledWith(pluginId, 'isReady', true);
+  // });
```

Comment:
Do we have to keep all this commented code ?

Reply:
Yes